nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -235,6 +235,11 @@ language governing permissions and limitations under the License. -->
             <artifactId>json-utils</artifactId>
             <version>0.0.20</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.calcite</groupId>
+            <artifactId>calcite-example-csv</artifactId>
```

Comment:
"We don't want multiple CSV libraries, do we?"
I agree it is ideal to avoid multiple libraries that 'do the same thing'.

However...

What we really favor is much more than just that.  Bringing in something like Calcite's libraries means we're adopting their design decisions which they made out of good faith and effort just as we do with internally developed things.  These cases cause natural tension on transitive dependencies.  This is a great example of why we've provided the classloader isolation model we have in NiFi.  https://en.wiktionary.org/wiki/let_a_thousand_flowers_bloom